modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -258,7 +236,7 @@ TEST(TestAgeGenderIE, InferROIList)
     cv::GComputation comp(cv::GIn(in, rr), cv::GOut(age, gender));
 
     auto pp = cv::gapi::ie::Params<AgeGender> {
-        topology_path, weights_path, "CPU"
+        params.model_path, params.weights_path, params.device_id
     }.cfgOutputLayers({ "age_conv3", "prob" });
     comp.apply(cv::gin(in_mat, rois), cv::gout(gapi_age, gapi_gender),
                cv::compile_args(cv::gapi::networks(pp)));
```

Comment:
A test which checks a two-networks-inference-in-a-row pipeline should be implemented to avoid potential single IE::Core issues